aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -30,21 +30,29 @@ The sample Dockerfile uses the [Docker multi-stage build feature](https://docs.d
 
 * `dotnet/core/sdk`
```

Comment:
This should be `dotnet/sdk` for 5.0+. Also, in the "In this tutorial" section above, change "Learn about Microsoft .NET Core Docker images" to "Learn about ASP.NET Core Docker images".